SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -580,6 +582,51 @@ void USpatialNetDriver::CreateAndInitializeCoreClassesAfterStartup()
 		}
 
 		ClientConnectionManager = MakeUnique<SpatialGDK::ClientConnectionManager>(SystemEntitySubview, this);
+
+		if (IsServer())
```

Comment:
There's a few places where it looks like we can reduce nesting if we invert some if statements and early out.

Reply:
However if this is just copying stuff over then it's not a big deal to fix this here.